apps/system/utils/Kconfig

```diff
@@ -158,7 +158,10 @@ config ENABLE_KILL_CMD
 	default y
 	depends on !BUILD_PROTECTED
 	---help---
-		Send a signal to processes or process groups
+		Send a signal to processes or process groups.
+		When SIGKILL(9) is sent, a task/pthread which received signal will be terminated without any garbage collection.
+		For freeing the allocated memory, user's own handler for SIGKILL is needed.
```

Comment:
How about 'user needs to add own handler to free the allocated(or used) memory'?